### ISSUE TITLE
Is 8bit supported?

### ISSUE THREAD

**Issue by nivibilla** [mit-han-lab/qserve#2]
Does this also work for 8bit models? Or only 4bit.

**Comment by kentang-mit** [comment]
Yes. W8A8KV8/KV4 runtime is implemented in this repo. We are also working on the model converter to convert LMQuant-W8A8 checkpoints to QServe format.

**Comment by nivibilla** [comment]
Amazing thank you! Will wait for that.

**Comment by ys-2020** [comment]
Hi @nivibilla , we have prepared the scripts for W8A8 inference. Please refer to #4 . Thanks!
